src/ssh/test-sshbridge.c

```diff
@@ -849,6 +853,40 @@ static const TestFixture fixture_home_ssh_config_invalid_user = {
   .problem = "authentication-failed"
 };
 
+static const TestFixture fixture_home_ssh_config_good_key = {
+  .knownhosts_file = "/dev/null",
+  .test_home_ssh_config = TRUE,
+  .ssh_config_user = USER_ME,
+  .ssh_config_identity_file = SRCDIR "/src/ssh/test_rsa",
+  .client_password = "bad password", /* with a bad password we'll have to rely on pubkey auth */
+  .knownhosts_home = MOCK_RSA_KEY,
+  .allow_unknown = TRUE,
+  .ssh_command = BUILDDIR "/mock-echo",
+};
+
+static const TestFixture fixture_home_ssh_config_bad_key = {
+  .knownhosts_file = "/dev/null",
+  .test_home_ssh_config = TRUE,
+  .ssh_config_user = USER_ME,
+  .ssh_config_identity_file = SRCDIR "/src/ssh/mock_rsa_key",
+  .client_password = "bad password", /* with a bad password we'll have to rely on pubkey auth */
```

Comment:
can you make the comment more like "this will cause a failure because the key was also wrong"